modules/ossm-kiali-viewing-metrics.adoc

```diff
@@ -0,0 +1,36 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+[id="ossm-viewing-logs_{context}"]
+= Viewing logs in the Kiali console
+
+You can view logs for your workloads in the Kiali console.  The *Workload Detail* page includes a *Logs* tab which displays a unified logs view that displays both application and proxy logs.
+
+To change the logging level on the logs displayed in Kiali, you change the logging configuration for the workload or the proxy.
+
+//How often are logs updated?
+// Kiali can also add-in trace span information to help identify important traces based on associated logging. More powerful features include substring or regex Show/Hide, full-screen, and the ability to set proxy log level without a pod restart.
```

Comment:
The trace add-in will be in 2.2.  Also in 2.2, the ability to set proxy-log-level w/o restart.  The show/hide and full-screen is in 2.1.  Also in 2.1, not mentioned here, is the ability to see parsed access log detail, see `Enhanced Logs tab` section in https://medium.com/kialiproject/kiali-releases-1-31-to-1-33-overview-6b18e23a06e2.

Reply:
Since these aren't currently available I'll create JIRAs to add docs for these features when they're released.